src/rpc/core_rpc_server.cpp

```diff
@@ -163,6 +163,8 @@ namespace cryptonote
     command_line::add_arg(desc, arg_rpc_payment_difficulty);
     command_line::add_arg(desc, arg_rpc_payment_credits);
     command_line::add_arg(desc, arg_rpc_payment_allow_free_loopback);
+    command_line::add_arg(desc, arg_rpc_max_ip_connections);
```

Comment:
We have `max-connections-per-ip` for p2p.
probably best to remain consistent and use `rpc-max-connections-per-ip`

Reply:
Updated in force push.